### PR TITLE
Run fuzzers on macOS arm64

### PR DESCRIPTION
**Pull request by jakobbotsch**
c.f. #67274

### CODE REVIEW

**Comment by jakobbotsch** [comment]
Many examples found (scroll down to "Summary of Fuzzlyn run on OSX arm64 checked"): https://dev.azure.com/dnceng/public/_build/results?buildId=1688568&view=ms.vss-build-web.run-extensions-tab


Hopefully most of these are fixed by #67274.

File: src/coreclr/scripts/fuzzer_setup.py [hunk 1]
```diff
@@ -140,7 +143,7 @@ def main(main_args):
     except PermissionError as pe:
         print("Skipping file. Got error: %s", pe)
 
-    # create foo.txt in work_item directories
+    # create a file in the in work_item directories, otherwise Helix complains
```
Comment (kunalspathak) [inline src/coreclr/scripts/fuzzer_setup.py:146]:
```suggestion
    # create a dummy file in the work_item directories, otherwise Helix complains
```

**Review by kunalspathak — APPROVED** [review]
LGTM

**Pull request merged**